app/channels/application_cable/channel.rb

```diff
@@ -0,0 +1,4 @@
+module ApplicationCable
+  class Channel < ActionCable::Channel::Base
+  end
+end
```

Comment:
I don't see any changes being done for this, making it a useless class for now, we should consider removing it for now as it has no use, WDYT?